scripts/testnet/sovereignBridge/config/esdt-safe.snippets.sh

```diff
@@ -0,0 +1,168 @@
+ESDT_SAFE_ADDRESS=$(mxpy data load --partition=${CHAIN_ID} --key=address-esdt-safe-contract)
+ESDT_SAFE_ADDRESS_SOVEREIGN=$(mxpy data load --partition=sovereign --key=address-esdt-safe-contract)
+
+deployEsdtSafeContract() {
+    mxpy --verbose contract deploy \
+        --bytecode=$(eval echo ${ESDT_SAFE_WASM}) \
+        --pem=${WALLET} \
+        --proxy=${PROXY} \
+        --chain=${CHAIN_ID} \
+        --gas-limit=200000000 \
+        --arguments \
+            false \
+            ${MIN_VALID_SIGNERS} \
+            ${INITIATOR_ADDRESS} \
+            ${SIGNERS} \
+        --outfile="${SCRIPT_PATH}/deploy-esdt-safe.interaction.json" \
+        --recall-nonce \
+        --wait-result \
+        --send || return
+
+    local TX_STATUS=$(mxpy data parse --file="${SCRIPT_PATH}/deploy-esdt-safe.interaction.json"  --expression="data['transactionOnNetwork']['status']")
```

Comment:
I would say the 'status' will never be deleted from fields...
The rest of the checks are processed only if the status is success, but those can panic as well.
Anyways this is not really a problem because you cannot do the transactions and stuff if you don't have the addresses for contracts